### ISSUE TITLE
MHP Juggernauts

### ISSUE THREAD

**Issue by JacobJeaurond** [torilmud/issues#971]
We did MHP last night and noticed this text when fighting the Juggernauts,  and it removed the para effects but it didn't seem to last as long as before.

Here is what I seen on screen:
#cLYou watch in #cwawe #cLand #cmhorror #cLas a massive juggernaut #cLrepairs itself.#N
#cLThe battle is gsnot yet won!#N

Thanks
Jacob (Lithius)

**Comment by GeneralGormal** [comment]
Not sure it's clear what the bug is here, the message or the para itself. The juggernauts can be kept from paralyzing by bash or blind I forget which btw!

**Comment by GeneralGormal** [comment]
Oh, I think this is related to the bug I posted about zone echoes not working right.  I think there's something that changed with color codes that these didn't get updated to.

**Comment by JacobJeaurond** [comment]
Maybe. But when I was soloing them the para would last at least 2x longer. He just couldnt hit you if blind and clr. But I got that text code and I was out of the para effect. Just wanted to mention that something looked off. Or we 2maned with lots of luck



Sent from my iPhone

On May 15, 2023, at 2:02 AM, GeneralGormal ***@***.***> wrote:

﻿

Oh, I think this is related to the bug I posted about zone echoes not working right. I think there's something that changed with color codes that these didn't get updated to.
